core/dbt/task/base.py

```diff
@@ -169,7 +170,7 @@ def get_nearest_project_dir(args):
 
 
 def move_to_nearest_project_dir(args):
-    nearest_project_dir = get_nearest_project_dir(args)
+    nearest_project_dir = get_nearest_project_dir(args.project_dir)
```

Comment:
since `get_nearest_project_dir` doesn't need `args` anymore, maybe we can also have `move_to_nearest_project_dir` just taking  `project_dir` as argument?

Reply:
yes yes! I actually thought I had that change in, not sure where it went. Will update